docs/snippets/Snippets.NUnit/MultipleAsserts.cs

```diff
@@ -0,0 +1,47 @@
+namespace Snippets.NUnit;
+
+public class MultipleAsserts
+{
+    #region MultipleAssertsProdCode
+    public class CalculationResult
+    {
+        public double RealPart { get; set; }
+        public double ImaginaryPart { get; set; }
+
+    }
+    public class SomeCalculator
+    {
+        public CalculationResult DoCalculation()
+        {
+            return new CalculationResult
+            {
+                // Hard-coded for demo
+                RealPart = 5.2,
+                ImaginaryPart = 3.9
+            };
+        }
+    }
+    #endregion
+
+    #region MultipleAssertsTests
+    [Test]
+    public void MultipleAssertsDemo()
+    {
+        var situationUnderTest = new SomeCalculator();
+        var result = situationUnderTest.DoCalculation();
+
+        Assert.Multiple(() =>
+        {
+            Assert.That(result.RealPart, Is.EqualTo(5.2));
+            Assert.That(result.ImaginaryPart, Is.EqualTo(3.9));
+        });
+
+        // Can also work with the classic assertion syntax
+        Assert.Multiple(() =>
+        {
+            Assert.AreEqual(5.2, result.RealPart, "Real Part");
```

Comment:
ℹ️ This generates a warning which is showing up in our PR. I'll have to find a way to disable those when we're explicitly doing that. Won't be a problem, just a note to self.